warehouse/forms.py

```diff
@@ -75,6 +83,24 @@ def __call__(self, form, field):
             raise ValidationError(msg)
 
 
+class PreventHTMLTagsValidator:
+    """
+    Validate the field to ensure that it does not contain any HTML tags.
+    """
+
+    def __init__(self, message: str | None = None):
+        if message is None:
+            message = "HTML tags are not allowed"
+        self.message = message
+
+    def __call__(self, form: BaseForm, field: Field):
+        # Override the default nh3.ALLOWED_TAGS to be an empty set
+        allowed_tags: set[str] = set()
+
+        if is_html(field.data) and field.data != clean(field.data, tags=allowed_tags):
```

Comment:
Won't the second comparison always fail if `is_html` is true?

Reply:
Seems like this could just be
```suggestion
        if is_html(field.data):
```

Reply:
That's correct. I'll remove the second condition, and this can be how we'd implement if if we _did_ want to allow any tags in the future.